geonetwork/transform-mcp-iso19115-3/schema/iso19115-3/process/transform-19115-3-testing-urls.xsl

```diff
@@ -48,13 +51,21 @@
 
   <xsl:template mode="substitute" match="@*|text()">
     <xsl:variable name="url" select="."/>
- 
     <xsl:for-each select="$urlSubstitutions/substitution">
       <xsl:if test="matches($url, string(@match))">
         <xsl:value-of select="replace($url, string(@match), string(@replaceWith))"/>
       </xsl:if>
     </xsl:for-each>
   </xsl:template>
 
+  <xsl:template match="mri:abstract/gco:CharacterString">
```

Comment:
Perhaps use urlSubstitutionSelector to only match mri:abstract elements containing a match